### ISSUE TITLE
Manual mode not working since Big Sur

### ISSUE THREAD

**Issue by dear-leader** [alin23/Lunar#238]
# Issue details
- Was the diagnostics process able to change the brightness on your external monitor(s)?
    Yes

- Mac device where Lunar is installed (Macbook Pro 2019, iMac, Mac Mini, Hackintosh etc.):
    MacBook Pro 15" A1990 2019

- Monitor connection to the Mac device (HDMI-to-USB-C, USB-C-to-USB-C, miniDisplayPort-to-DisplayPort etc.):
    DisplayPort to USB-C

- Using an USB Docking Station or Hub:
    No

- Lunar mode used (check it in the top-right corner of the Lunar interface)
    Manual

- (only if you know how to compile a C program) Does this utility work for you? https://github.com/kfix/ddcctl
    *optional response*

# Issue description:
Lunar was working perfectly with the same hardware setup when I had Catalina installed, but since installing Big Sur, Lunar cannot change the brightness of the two external displays when in Manual mode and using the keyboard's brightness keys.

The external display brightness can still be manually changed from the Lunar icon in the macOS menu bar, but using the brightness keys on the keyboard do not change the brightness of the active monitor (the option to "Use media keys as hotkeys **is** enabled in the Advanced menu). The brightness up/down keys only adjust the laptop's built in display brightness, regardless of which monitor the cursor's in. I've tried using the brightness keys both from the laptop's Touch Bar as well as with an Apple bluetooth keyboard with the same results.

I've since switched to using Sync mode, which is adjusting the brightness of the external displays as expected, but I'd prefer to use Manual mode.

**Comment by alin23** [comment]
This is most likely a permissions issue caused by the OS upgrade. Go into System Preferences->Security and Privacy and check if Lunar is enabled under the Accessibility and Input Monitoring permissions. If it is already, uncheck and recheck the checkbox.

Lunar needs these permissions in order to listen globally to the brightness and volume keys.

**Comment by dear-leader** [comment]
You called it. I'd actually checked Lunar's permissions in the Accessibility section already, but it superficially appeared to be enabled and I left it at that.

When I just tried to disable and reenable it per your advice, I found that the checkbox couldn't be unchecked. Other apps in the Accessibility section could have their permissions disabled/enabled as expected, but Lunar's checkbox wouldn't budge. I had to remove Lunar's permissions entry altogether and re-add it. And now Lunar's behaving normally again.

Thanks for the quick fix.
